instrumentation/opentelemetry-instrumentation-requests/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   ([#1040](https://github.com/open-telemetry/opentelemetry-python/pull/1040))
 - Drop support for Python 3.4
   ([#1099](https://github.com/open-telemetry/opentelemetry-python/pull/1099))
+- Add support for standard metrics - duration
```

Comment:
rename to something other than "standard metrics".
Add support for http metrics from Requests instrumentation or similar.